contracts/contracts/eosio.system/src/voting.cpp

```diff
@@ -86,6 +86,10 @@ namespace eosiosystem {
    void system_contract::unregprod( const name& producer ) {
       require_auth( producer );
 
+      const auto voter = _voters.find( producer.value );
```

Comment:
we can write this as one line:
const auto& voter = _voters.get(producer.value, "producer is not found in voter table");